src/Text/index.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+import * as React from "react";
+import * as tokens from "orbit-design-token";
+
+const colorText = {
+  primary: tokens.colorTextPrimary,
+  secondary: tokens.colorTextSecondary,
+  attention: tokens.colorTextAttention,
+};
+
+const weightText = {
+  regular: tokens.fontWeightNormal,
+  bold: tokens.fontWeightBold,
+};
+
+const sizeText = {
+  large: tokens.fontSizeTextLarge,
+  normal: tokens.fontSizeTextNormal,
+  small: tokens.fontSizeTextSmall,
+};
+
+type Props = {
+  type: $Keys<typeof colorText>,
+  size: $Keys<typeof sizeText>,
+  weight: "bold" | "regular",
+  align: "left" | "center" | "right",
+  isItalic?: boolean,
+  isUppercase?: boolean,
+  element: "p" | "span" | "div",
```

Comment:
is element in this component necessary ?

Reply:
It's written in the specification of Text component -> https://github.com/kiwicom/orbit-components/issues/91